Code/GraphMol/RGroupDecomposition/Wrap/rdRGroupComposition.cpp

```diff
@@ -200,25 +202,31 @@ struct rgroupdecomp_wrapper {
 
     python::enum_<RDKit::RGroupCoreAlignment>("RGroupCoreAlignment")
         .value("None", RDKit::None)
+        .value("None_", RDKit::None)
```

Comment:
Let's just do a breaking change here and switch it to NoAlignment both ini C++ and in Python.

I never should have used None in the first place ;)